sdk/core/src/tasks/enrichments/identity/cpe/sync.rs

```diff
@@ -0,0 +1,147 @@
+use std::collections::HashMap;
```

Comment:
I like this namespace/module name.

Reply:
Thank you!  It beats the hell out of "build".  That was my first thought, but I'm glad you like the naming.  I expected to have a discussion about it because I didn't consult you at first.